docs/development/reviewing-patches.rst

```diff
@@ -43,9 +43,9 @@ devastating, ``cryptography`` has a strict merge policy for committers:
 
 * Patches must *never* be pushed directly to ``master``, all changes (even the
   most trivial typo fixes!) must be submitted as a pull request.
-* A committer may *never* merge their own pull request, a second party must
-  merge their changes. If multiple people work on a pull request, it must be
-  merged by someone who did not work on it.
+* A committer may only merge their own pull request if it has been approved
+  through the GitHub UI by a second party. If multiple people work on a pull
```

Comment:
I think the rules should require that the approver also be a committer, which currently this doesn't.